JS-Drum-Kit/app.js

```diff
@@ -0,0 +1,32 @@
+const keys = document.querySelectorAll('.key');
+
+// Play drum by keyboard press
+
```

Comment:
you can create a function like
```suggestion
function play(keyCode) {
	const audio = document.querySelector(`audio[data-key="${keyCode}"]`);
	if(!audio) return;
	audio.currentTime = 0;
    audio.play();
    const key = document.querySelector(`.key[data-key="${keyCode}"]`);
    key.classList.add('playing');
}
```

Reply:
done